### PR TITLE
finished my part of player

### PR DESCRIPTION
**Pull request by AlexWIllini**

### CODE REVIEW

File: Player.cs [hunk 1]
```diff
@@ -2,14 +2,28 @@
 
 public class Player
 {
+    private List<Army> armyList = new List<Army>();
     public List<Army> GetArmies()
     {
-        return null; // TODO: Change
+        int empty = 0;
+        int isEmpty = armyList.Count;
+        if (isEmpty == empty)
```
Comment (shmasmasnash) [inline Player.cs]: Just return the list even if empty

**Review by shmasmasnash — CHANGES_REQUESTED** [review]
Remove the null return block

**Comment by vassilmladenov** [comment]
Thanks, please clean up the last of the style issues, then squash all your commits into one with an interactive rebase.

**Comment by AlexWIllini** [comment]
Why would I have to squash all my commits into one if I'm just committing to my local branch? Wouldn't it just show up as a single commit when the pull request gets accepted by you for the upstream master?

**Review by shmasmasnash — APPROVED** [review]

**Comment by vassilmladenov** [comment]
To answer your question, no, because that's not how rebase works. If we were only using merge commits, then yes, it would create a single commit here. What this does is it replays the commits of your feature branch onto master so that you get one continuous stream of history.

**Pull request merged**